src/spoke/BalanceSheet.sol

```diff
@@ -128,6 +128,22 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         escrow_.authTransferTo(asset, tokenId, receiver, amount);
     }
 
+    /// @inheritdoc IBalanceSheet
+    function reserve(PoolId poolId, ShareClassId scId, address asset, uint256 tokenId, uint128 amount)
+        public
+        authOrManager(poolId)
+    {
+        escrow(poolId).reserve(scId, asset, tokenId, amount);
+    }
+
+    /// @inheritdoc IBalanceSheet
+    function unreserve(PoolId poolId, ShareClassId scId, address asset, uint256 tokenId, uint128 amount)
+        public
+        authOrManager(poolId)
+    {
+        escrow(poolId).unreserve(scId, asset, tokenId, amount);
+    }
+
```

Comment:
@hieronx why do we need those? We had this logic already beforehand and agreed on not needing it.